case-lib/lib.sh

```diff
@@ -155,3 +155,22 @@ func_lib_lsof_error_dump()
         echo "$ret"
     fi
 }
+
+func_lib_get_tplg_path()
+{
+    local tplg=$1
+    # tplg given is empty
+    if [[ -z "$tplg" ]]; then
+        return 1
+    fi
+
+    if [[ -f "$tplg" ]]; then
+        echo "$tplg"
+    elif [[ -f "$TPLG_ROOT/$(basename $tplg)" ]]; then
```

Comment:
better to use "$TPLG_ROOT/$(basename "$tplg")"
in case tplg name includes blank space (although this shouldn't happen so far)

Reply:
the first `"` doesn't match the second `"`, and it matches to the forth `"`,  works like a magic. thanks, fixed.